ludwig/encoders/image_encoders.py

```diff
@@ -14,23 +14,43 @@
 # limitations under the License.
 # ==============================================================================
 import logging
+import os
+from abc import abstractmethod
 from typing import Any, Dict, List, Optional, Tuple, Union
 
 import torch
+import torchvision.models as tvm
 
 from ludwig.constants import IMAGE
 from ludwig.encoders.base import Encoder
 from ludwig.encoders.registry import register_encoder
-from ludwig.modules.convolutional_modules import Conv2DStack, ResNet
+from ludwig.modules.convolutional_modules import Conv2DStack
 from ludwig.modules.fully_connected_modules import FCStack
 from ludwig.modules.mlp_mixer_modules import MLPMixer
 from ludwig.schema.encoders.image_encoders import (
     MLPMixerEncoderConfig,
-    ResNetEncoderConfig,
     Stacked2DCNNEncoderConfig,
-    ViTEncoderConfig,
+    TVAlexNetEncoderConfig,
```

Comment:
In order to keep the `image_encoders` source file size under control, I might suggest moving these new registrations to a separate file, i.e. `torchvision_image_encoders.py` or `image_torchvision_encoders.py`.

Reply:
moved torchvision encoders to `image_torchvision_encoders.py` and adjusted related tests